package.json

```diff
@@ -12,13 +12,13 @@
     "test-accessibility-cli-vocab-search": "npx cypress run --env cli=true --spec 'tests/cypress/accessibility/axe-vocab-search.cy.js'"
   },
   "dependencies": {
-    "bootstrap": "^5.2.3",
-    "vue": "^3.2.47"
+    "bootstrap": "^5.3.x",
```

Comment:
Very minor remark: Using `x` here looks a bit weird to me, although it seems to be a valid convention.

We are using `*` in composer.json, would it be good to be consistent and use it here as well?

Reply:
Will do!